README.md

```diff
@@ -1 +1,188 @@
-# cloudant-spring
+# Cloudant Spring Library
+
+The is a library to provide Spring developers easy configuration of the [official Cloudant library for Java](https://github.com/cloudant/java-cloudant/).
+
+The library is split into two parts:
+* `cloudant-spring-boot-starter` for [Spring Boot](https://projects.spring.io/spring-boot/) applications
+
+* `cloudant-spring-framework` for [Spring Framework](https://projects.spring.io/spring-framework/) applications
+
+
+* [Installation and Usage](#installation-and-usage)
+* [Getting Started](#getting-started)
+* [Related Documentation](#related-documentation)
+* [Development](#development)
+    * [Contributing](CONTRIBUTING.md)
+    * [Test Suite](CONTRIBUTING.md#running-the-tests)
+    * [Using in Other Projects](#using-in-other-projects)
+    * [License](#license)
+    * [Issues](#issues)
+
+## Installation and Usage
+
+### Spring Boot Applications
+
+Gradle:
+```groovy
+dependencies {
+    compile group: 'com.cloudant', name: 'cloudant-spring-boot-starter', version: '0.0.1'
+}
+```
+
+Maven:
+~~~ xml
+<dependency>
+  <groupId>com.cloudant</groupId>
+  <artifactId>cloudant-spring-boot-starter</artifactId>
+  <version>0.0.1</version>
+</dependency>
+~~~
+
+### Spring Framework Applications
+
+Gradle:
+```groovy
+dependencies {
+    compile group: 'com.cloudant', name: 'cloudant-spring-framework', version: '0.0.1'
+}
+```
+
+Maven:
+~~~ xml
+<dependency>
+  <groupId>com.cloudant</groupId>
+  <artifactId>cloudant-spring-framework</artifactId>
+  <version>0.0.1</version>
+</dependency>
+~~~
+
+## Getting Started
+
+This section contains simple examples of connecting to Cloudant using the two libraries.
+
+### Spring Boot Applications
+
+You must provide the following properties:
```

Comment:
suggest appending "to define the connection to your Cloudant instance" since you refer to "your Cloudant instance" later. It ought to be pretty obvious, but no harm in extra clarity I think.